src/inventory.go

```diff
@@ -76,7 +78,13 @@ func populateInventory(reader *bufio.Reader, i *inventory.Inventory) error {
 		case '#':
 			// ignore comments
 			for r != '\n' {
-				r, _, _ = reader.ReadRune()
+				r, _, err = reader.ReadRune()
```

Comment:
can we add a new test covering the old bug?

Reply:
I tried it but seems that reading it from a string like `testNginxConf` I can't reproduce it...  I'll think of another way of testing it then.

Reply:
I end up doing it straight from a file here 8bdb34e7c669a954122e1fbf62e4af741535deb3.